docs/intro/transparency.md

```diff
@@ -0,0 +1,48 @@
+*NiPreps* adopt *fMRIPrep*'s foundations, and particularly resonate with the transparency principles.
+As discussed in ([Esteban et al., 2019](https://doi.org/10.1038/s41592-018-0235-4) -- [preprint](https://doi.org/10.1101/306951)):
+
+> The rapid increase in the volume and diversity of data, as well as the evolution of available techniques for processing and analysis, presents an opportunity for considerable advancement of research in neuroscience.
+> The drawback resides in the need for progressively more complex analysis workflows that rely on decreasingly interpretable models of the data.
+> Such context encourages ‘*black-box*’ solutions that efficiently perform a valuable service but do not provide insights into how the tool has transformed the data into the expected outputs.
+> Black boxes obscure important steps in the inductive process mediating between experimental measurements and reported findings.
+> This way of moving forward risks producing a future generation of cognitive neuroscientists who have become experts in sophisticated computational methods but have little to no working knowledge of how their data were transformed through processing.
+> Transparency is often identified as a remedy for these problems.
+> *fMRIPrep* ascribes to ‘*glass-box*’ principles, which are defined in opposition to the many different facets or levels at which black-box solutions are opaque.
+> The visual reports that *fMRIPrep* generates are a crucial aspect of the glass-box approach.
+> Their quality control checkpoints represent the logical flow of preprocessing, allowing scientists to critically inspect and better understand the underlying mechanisms of the workflow.
+> A second transparency element is the citation boilerplate that formalizes all details of the workflow and provides the versions of all involved tools along with references to the corresponding scientific literature.
+> A third asset for transparency is thorough documentation that delivers addi-tional details on each of the building blocks represented in the visual reports and described in the boilerplate.
```

Comment:
no need for "addi-tional".  "additional" should be fine here.

Reply:
```suggestion
> A third asset for transparency is thorough documentation that delivers additional details on each of the building blocks represented in the visual reports and described in the boilerplate.
```